packages/lib/passport-interface/src/genericIssuanceTypes.ts

```diff
@@ -165,7 +185,16 @@ const LemonadePipelineOptionsSchema = BasePipelineOptionsSchema.extend({
   events: z.array(LemonadePipelineEventConfigSchema),
   superuserEmails: z.array(z.string()).optional(),
   feedOptions: FeedIssuanceOptionsSchema,
-  manualTickets: z.array(ManualTicketSchema).optional()
+  manualTickets: z.array(ManualTicketSchema).optional().default([]),
+  semaphoreGroups: z
+    .array(SemaphoreGroupConfigSchema)
+    .optional()
+    .default([])
```

Comment:
This turns the inferred type `SemaphoreGroupConfig[]`, which accurately reflects what the parsed output would be, but might be dangerous if used as part of a type assertion on JSON loaded from the DB without parsing.